README.md

```diff
@@ -2,7 +2,7 @@
 
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2FFreeFeed%2Ffreefeed-react-client.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2FFreeFeed%2Ffreefeed-react-client?ref=badge_shield)
 
-[Node.js](https://nodejs.org) 14 or 16 is supported.
+[Node.js](https://nodejs.org) 18 is supported.
```

Comment:
It's better to base on our [test actions](https://github.com/FreeFeed/freefeed-react-client/blob/stable/.github/workflows/checks.yml) rather than on the Dockerfile (which can only have one version). We, actually, supporting Node 16, 18 and 20.

Reply:
fixed